ipatests/pytest_plugins/integration/tasks.py

```diff
@@ -1314,7 +1314,8 @@ def ldappasswd_user_change(user, oldpw, newpw, master):
     basedn = master.domain.basedn
 
     userdn = "uid={},{},{}".format(user, container_user, basedn)
+    master_ldap_uri = "ldap://{}:389".format(master.external_hostname)
```

Comment:
nit: is the 389 really necessary? It is the default when no port is used for an ldap uri.